include/swift/SILOptimizer/PassManager/Passes.def

```diff
@@ -179,6 +179,12 @@ PASS(PredictableMemoryOptimizations, "predictable-memopt",
      "Predictable early memory optimizations")
 PASS(ReleaseDevirtualizer, "release-devirtualizer",
      "Devirtualize release-instructions")
+PASS(RetainSinking, "retain-sinking",
```

Comment:
I need to read through the rest of the passes, but I also wonder if this can be done in one pass instead of three.